content/projects/gsoc/2019/plugin-installation-manager-tool-cli.adoc

```diff
@@ -36,15 +36,22 @@ There are currently many reincarnations of plugin management across Jenkins, inc
 
 This project will create a new Plugin Manager CLI tools and a library which would unify the plugin management across different Jenkins implementations.
 Although we looked at extracting plugin management from Jenkins Core into its own module, this proved to be a challenging first step (see link:https://issues.jenkins-ci.org/browse/JENKINS-57672[JENKINS-57672]).
-Right now the project is focusing on extracting the link:https://github.com/jenkinsci/docker/blob/master/install-plugins.sh[docker install-plugins bash script] to Java.
+Phase 1 focused on extracting the link:https://github.com/jenkinsci/docker/blob/master/install-plugins.sh[docker install-plugins bash script] to Java.
+Phase 2 will include improving upon what was written in Phase 1 and full integration with link:https://github.com/jenkinsci/docker[Jenkins Docker].
```

Comment:
Anything more specific here other than improving and docker integration?

Reply:
I don't think we've come to a conclusion on the full scope yet, but this should be worked out in the next few days as part of the planning process for Phase 2.